arrowhead/arrowhead.rb

```diff
@@ -1,5 +1,4 @@
 class Arrowhead
-  # This seriously belongs in a database.
```

Comment:
This comment probably still belongs here since we didn't actually fix this problem.